.github/workflows/pre-commit.yml

```diff
@@ -38,33 +73,64 @@ jobs:
       run: |
         python -m pytest src/test/python
 
-  lint:
-    name: Lint ${{ matrix.package }}
+  lintpython27:
+    name: Lint ${{ matrix.package }} - Python27
 
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 4
       matrix:
-        package: [tools]
-        
+        package: [common, libs, platform]
```

Comment:
any reason why we're not also doing tools here?

Reply:
`tools` is py3 only now